src/sentry/static/sentry/app/views/organizationDashboard/data/queries/eventsByRelease.jsx

```diff
@@ -5,14 +5,71 @@ import {t} from 'app/locale';
 
 const eventsByRelease = {
   name: t('Events by Release'),
-  fields: ['release'],
+  fields: ['__release'],
```

Comment:
This is required for the frontend when parsing results -- we don't forward this when making discover query. Currently, we need to declare it here because this query gets passed around when generating results.